src/OpenConext/ProfileBundle/Resources/views/MyServices/overview.html.twig

```diff
@@ -67,7 +67,7 @@
                                 </tr>
                                 </thead>
                                 <tbody>
-                                {% for attribute in app.user.attributes %}
+                                {% for attribute in consent.attributes %}
```

Comment:
(hence my preference for a new different object for consent with attributes, so we known it always returns an iterable)